docs/api/ReactWrapper/isEmptyRender.md

```diff
@@ -0,0 +1,18 @@
+# `.isEmptyRender() => Boolean`
+
+Returns whether or not the current component returns a falsey value: `false`, `null`, `undefined`.
```

Comment:
(altho actually i'll just cherry-pick this into master now and fix it inline; you can rebase after that and omit this commit)